uber/templates/macros.html

```diff
@@ -119,50 +119,51 @@
   </div>
 {%- endmacro %}
 
-{% macro address_form(group=False, attendee=False, is_prereg_dealer=False, copy_address=False, name_prefix="") -%}
+{% macro address_form(group=False, attendee=False, is_dealer=False, collect_full_address=False, copy_address=False, name_prefix="") -%}
     {% set group_or_attendee=group if group else attendee %}
     {% set id_prefix=group.id|replace("-","") if group else attendee.id|replace("-","") %}
-    {% if not is_prereg_dealer %}
+    {% if is_dealer or collect_full_address %}
         <script src="../static/deps/selectToAutocomplete/jquery.select-to-autocomplete.js"></script>
         <script type="text/javascript">{% include "region_opts.html" %}</script>
-    {% endif %}
-    <script type="text/javascript">
```

Comment:
I just moved this inside the `{% if is_dealer or collect_full_address %}` conditional with the other full address `<script>` tags.